LEAF_Request_Portal/js/formGrid.js

```diff
@@ -45,7 +45,14 @@ var LeafFormGrid = function(containerID, options) {
      * @param values (required) object of cells and names to generate grid
      * @memberOf LeafFormGrid
      */
-    function printTableReportBuilder(values) {
+    function printTableReportBuilder(values, columnValues) {
+        // remove unused columns
+        if (columnValues !== null) {
+            values.format = values.format.filter(function (value) {
+                return columnValues.includes(value.id);
+            });
+        }
+
```

Comment:
I get an error here if I try to edit/save the grid section.  Adding && !== undefined will fix it.  After editing, the grid resets to the full grid (I don't know if that's an issue or not.  The dialog boxes are their own little monsters).